registrations/models.py

```diff
@@ -328,6 +329,10 @@ class SignUpGroup(CreatedModifiedBaseModel, SignUpMixin, SerializableMixin):
     registration = models.ForeignKey(
         Registration, on_delete=models.PROTECT, related_name="signup_groups"
     )
+    anonymization_time = models.DateTimeField(
+        verbose_name=_("Anonymization time"),
+        null=True,
+    )
```

Comment:
```suggestion
    anonymization_time = models.DateTimeField(
        verbose_name=_("Anonymization time"),
        null=True,
        editable=False,
    )
```
I think it would be safer to have `editable=False` here to prevent editing the value via Django admin.

Reply:
Good point. Added `editable=False` to `anonymization_time` field